assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -202,9 +202,16 @@ export const createSettingsStore = ( type, identifier, datapoint, {
 		 * @return {Object} Redux-style action.
 		 */
 		*fetchSaveSettings( values ) {
-			invariant( values, 'values is required.' );
 			let response, error;
 
+			if ( ! values ) {
+				error = {
+					message: 'values is required.',
+				};
+				global.console.error( error.message );
+				return { response, error };
+			}
```

Comment:
Shouldn't we check values in `actions.saveSettings` with invariant instead?

Reply:
I think it would be better to check for validity in `fetch*` since that's what actually needs the value. However, there was a problem with using `invariant` in there which I've already discussed a few weeks ago with @tofumatt. For some reason it never threw the error, the code would just continue as if all was fine. We agreed to for now use `console.error`.